hypertrace-trace-enricher/enriched-span-constants/src/main/java/org/hypertrace/traceenricher/enrichedspan/constants/utils/EnrichedSpanUtils.java

```diff
@@ -93,6 +94,9 @@
   private static final String OTEL_HTTP_USER_AGENT =
       OTelHttpSemanticConventions.HTTP_USER_AGENT.getValue();
 
+  private static final String API_TRACE_ERROR_COUNT =
+      EnrichedSpanConstants.getValue(ErrorMetrics.ERROR_API_TRACE_ERROR_COUNT);
```

Comment:
Do we usually prefix the constants based on the use case? Why not just call the constant as `API_TRACE_ERROR_COUNT`?